test/run-drun/gc-trigger.mo

```diff
@@ -0,0 +1,36 @@
+//MOC-NO-FORCE-GC
+//INCREMENTAL-GC-ONLY
```

Comment:
I wasn't sure why you need this INCREMENTAL-GC-ONLY flag..

Reply:
Keep if needed, I was just wondering if this was debugging residue.

Reply:
I only used it for the `gc-trigger.mo` test case (as the generational GC reclaimed memory earlier than I needed for test).